pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Ethical aspects
+contributors: [Karel Berka, Korbinian Bösl, Erin Calhoun, Paulette Lieby]
+description: Working on aspects in the management of research data that can raise ethical issues
+page_id: ethics
+related_pages:
+  tool_assembly: [covid-19, transmed, tsd, csc]
+training:
+- name: 'Ethics/ELSI considerations - From FAIR to fair data sharing - recorded webinar'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/ethics-elsi-considerations-from-fair-to-fair-data-sharing
+- name: 'Biomedical data: Ethical, legal and social implications - e-learning'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/biomedical-data
+- name: 'From Swab to Server: Testing, Sequencing, and Sharing During a Pandemic - free online course'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/from-swab-to-server-testing-sequencing-and-sharing-during-a-pandemic
+---
+​
+Ethical issues refer to moral principles and standards that guide human conduct and define what is considered right or wrong within a particular context.
+Ethics covers topics beyond the issue of personal data, compliance with legal obligations, and the collection and use of data.
```

Comment:
When it comes to research involving human participants, such ethical concerns may appear when accessing, using, or sharing data of a sensitive nature, for example health or personal data.  Ethics, however, goes beyond the issue of..

Reply:
```suggestion
When it comes to research involving human participants, such ethical concerns may appear when accessing, using, or sharing data of a sensitive nature, for example health or personal data. Ethics, however, goes beyond the issue of compliance with legal obligations, and the collection and use of data.
```